src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothError.cs

```diff
@@ -105,6 +105,88 @@ static internal void ThrowBluetoothException(int exception)
                 throw new InvalidOperationException("Unknown exception");
             }
         }
+
+        /// <summary>
```

Comment:
Please, remove duplication of code
How about refactoring 
`static internal void ThrowBluetoothException(int exception)` method with CreateBluetoothException?

``` c#
internal static void ThrowBluetoothException(int exception)
{
    throw CreateBluetoothException(exception);
}
```

Reply:
Done. Thank you.